README.org

```diff
@@ -1,24 +1,29 @@
 * helm icons
 
-  [[file:images/treemacs-icons.gif]]
+[[file:images/treemacs-icons.gif]]
 
 ** Configuration
 *** Melpa (TBD)
 *** Manual
-    Make sure that you have both =helm= and =treemacs= installed clone the repo
-    and put it on your path. Make sure that the enable call is performed before
-    loading =helm=.
+By default, the provider uses =all-the-icons=, so make sure that you have that
```

Comment:
can  you fix that line?

Reply:
Fix as in? Maintain the original indenting?

Reply:
The default is now treemacs.

Reply:
Oh yeah I can't read xD